WordPress/Docs/CodeAnalysis/AssignmentInTernaryConditionStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Assignment In Ternary Condition">
+<standard>
+    <![CDATA[
+        Checks that variable assignment does not occur in conditional statement of ternary. Condition must be in parentheses to be checked; Ternaries lacking parentheses around condition are skipped. If incorrect will throw: "Variable assignment found within a condition. Did you mean to do a comparison?"
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Variable assigned with comparison in parentheses.">
+        <![CDATA[
+echo $mode = ( <em>$a == 'a'</em> ) ? 'b' : 'c';
```

Comment:
Unless I'm missing something, I don't think that having an assignment and then echoing in the same line is necessary to illustrate a valid example and, in my opinion, adds a layer of complexity that makes it a bit harder to understand this example.

Reply:
This was a misunderstanding on my part from mirroring another file; removed